analyzers/its/expected/Ember-MM/Ember Media Manager-{9B57D3AB-AF12-4012-B945-284C2448DC81}-S1192.json

```diff
@@ -3791,7 +3791,7 @@
 },
 {
 "id":  "S1192",
-"message":  "Define a constant instead of using this literal '/select,""{0}""' 4 times.",
+"message":  "Define a constant instead of using this literal '/select,"{0}"' 4 times.",
```

Comment:
This is no longer valid jon

Reply:
LOL, that's a fail. I suspect this piece caused the issue:
https://github.com/SonarSource/sonar-dotnet/blob/8286a19c9a4c1974bd5cde3395e1c189a6aaca60/analyzers/its/create-issue-reports.ps1#L158-L163

It's never read as json AFAICT and compared only as text so it should work for it's purpose. Might be a thing to put into notes to investigate one day.

Reply:
I did a note. That unescape looks *interesting*.